### PR TITLE
Revamp First Input Timing section in explainer

### PR DESCRIPTION
**Pull request by npm1**
Currently this section is pretty hard to follow. This PR adds clarity and a code snippet. Part 1 to solving https://github.com/WICG/event-timing/issues/61

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,23 +84,40 @@ performanceObserver.observe({entryTypes:['event']});
...
 * Pointer down which is followed by a pointer up
 * Click
 
-This is ~4x the 99'th percentile of these events overall.
-In the median, we see ~10ms for the first event, and ~2.5ms for subsequent events.
+In Chrome, the 99'th percentile of FID is over 1 second.
+This is ~4x the 99'th percentile of the input delay of these events overall.
+In the median, we see a delay of ~10ms for the first event, and ~2.5ms for subsequent events.
 
 This list intentionally excludes scrolls, which are often not blocked on javascript execution.
```
Comment (anniesullie) [inline README.md]: Maybe move "This list intentionally excludes..." directly under the list?

File: README.md [hunk 2]
```diff
@@ -84,23 +84,40 @@ performanceObserver.observe({entryTypes:['event']});
...
 
-In order to address capture user pain caused by slow initial interactions, we propose always reporting first input timing within the event timing API specific to this use-case.
-      
-FirstInputDelay can be polyfilled today: see [here](https://github.com/GoogleChromeLabs/first-input-delay) for an example.
-However, this requires registering analytics JS before any events are processed, which is often not possible.
-First Input Delay can also be polyfilled on top of the event timing API, but it isn't very ergonomic, and due to the asynchrony of `performance.eventCounts` can sometimes incorrectly report an event as the first event when there was a prior event less than 100ms.
+In order to address capture user pain caused by slow initial interactions, we propose always reporting first input timing within the Event Timing API.
```
Comment (anniesullie) [inline README.md]: I know "address capture user pain" was in the original, but it's very confusing to me. Do we mean something like "understand and address user pain"?

File: README.md [hunk 3]
```diff
@@ -84,23 +84,40 @@ performanceObserver.observe({entryTypes:['event']});
...
+  // There should be a single entry.
+  const firstInput = entries().getEntries()[0];
+  const inputDelay = firstInput.processingStart - firstInput.startTime;
+  // Report inputDelay to analytics.
+  observer.disconnect();
+}).observe({type: 'first-input', buffered: true]});
+```
+
+FID can be polyfilled today: see [here](https://github.com/GoogleChromeLabs/first-input-delay) for an example.
+However, this requires registering analytics JS before any events are processed, which is often impossible or undesirable.
```
Comment (anniesullie) [inline README.md]: Not sure who the readership is here; do we need to be clearer about why it's impossible or undesirable for analytics JS to be registered before any events are processed? To me, it might be clearer to say "which is often impossible or undesirable for third party analytics providers, who do not have control over how their scripts are loaded and would like to be able to load them asynchronously to minimize their impact on page load speed."

File: README.md (hunk 1 above)
Comment (npm1) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (npm1) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (npm1) [reply README.md]: Good suggestion, added
